sklearn/model_selection/tests/test_validation.py

```diff
@@ -61,6 +61,7 @@
 from sklearn.datasets import make_multilabel_classification
 
 from sklearn.model_selection.tests.test_split import MockClassifier
```

Comment:
> Well if they are specific to a test, then I think they should stay there.

They are specific to 2 sets of test in 2 different test files, but not anywhere else except the `model_selection` tests, hence my disinclination to move them to `utils.testing` :)

Maybe you want them moved to `sklearn.model_selection.tests._test_setup`?